components/list/README.md

```diff
@@ -414,6 +413,7 @@ The `d2l-list-item` provides the appropriate `listitem` semantics for children w
 | `key` | String | Value to identify item if selectable or draggable |
 | `label` | String | Explicitly defined label for the element |
 | `labelled-by` | String | The id of element that provides the label for this element |
+| `selection-disabled` | Boolean | Disables selection |
```

Comment:
Because the `ListItemButtonMixin` extends `ListItemMixin`, `selection-disabled` is also added to the `list-item-button` so we should update that part of the README as well